src/core/orderbookutils.js

```diff
@@ -50,7 +85,10 @@ OrderBookUtils.getOfferTakerPaysFunded = function(offer) {
 OrderBookUtils.getOfferTakerGets = function(offer) {
   assert(typeof offer, 'object', 'Offer is invalid');
 
-  return Amount.from_json(offer.TakerGets + IOU_SUFFIX);
+  let currency = offer.TakerPays.currency;
+  let issuer = offer.TakerPays.issuer;
+
+  return Amount.from_json(createAmount(offer.TakerGets, currency, issuer));
```

Comment:
Not sure if it's a bug, or just poorly named. Taking a peek: https://github.com/ripple/ripple-lib/blob/5b4deabd90e5237bd0165e65520f25456d32a410/src/core/autobridgecalculator.js#L123